user_guide/common_features.rst

```diff
@@ -0,0 +1,79 @@
+.. _user-guide-common-features:
+
+Common Features
+===============
+
+Customers, Channels and Accounts
+--------------------------------
+
+One of the main tasks for any CRM is ability to aggregate the details of a customer. 
+For example, you have two web-based stores, a retail outlet and a factory. There is a company whose managers 
+purchase from your web-based stores and with whom you have some business arrangements at the 
+factory. 
+
+Of course, there are much more details and peculiarities, but if we keep it simple, in an on-line store, somebody comes 
+to the store's website, looks through the items offered, and if interested in 
+buying them puts the items in the cart and orders them, leaving you the contact and shipping details. In many cases the 
+web-store customers can log-in and their profile details will be used. What kind of information do you need to make 
+relations with such customers successful? The most basic things are - what pages have people viewed, what items they 
+have ordered, if there is anything that a customer has added to the cart but hasn't ordered and if there are some 
+complaints after the delivery.
```

Comment:
Edit: Of course, there are many more details and peculiarities to keep track of. However, if we keep it simple, a customer visits the store's website, looks through the items offered, and, if interested in buying them, puts the items in the cart and makes their order. In order to this, they leave you their contact and shipping details. In many cases, customers can login to use profile details they have already saved on your store. With this in mind, what kind of information do you need in order to have successful relations with your customers? The most basic factors to keep track of are what pages or products they have viewed, what items they have ordered, if there is anything that they have added to their cart but haven't ordered and if there are any complaints after the delivery.